# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsGlobalWindowInner.cpp
# Commit: 459739221a88
# Full Hash: 459739221a884b2b1c2adab42fd19fafe279bd20
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing build bustages
#   
#   This reverts commit ab681e36aeafc21ba62bb4680bdaa6206f02546d.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r fc9986230203 -r 459739221a88 dom/base/nsGlobalWindowInner.cpp
--- a/dom/base/nsGlobalWindowInner.cpp	Thu Jun 05 00:15:19 2025 +0300
+++ b/dom/base/nsGlobalWindowInner.cpp	Thu Jun 05 00:19:41 2025 +0300
@@ -183,7 +183,6 @@
 #include "mozilla/dom/VRDisplayEventBinding.h"
 #include "mozilla/dom/VREventObserver.h"
 #include "mozilla/dom/VisualViewport.h"
-#include "mozilla/dom/WebIdentityHandler.h"
 #include "mozilla/dom/WebIDLGlobalNameHash.h"
 #include "mozilla/dom/WindowBinding.h"
 #include "mozilla/dom/WindowContext.h"
@@ -7791,18 +7790,6 @@
   return wc->GetUsingStorageAccess();
 }
 
-WebIdentityHandler* nsPIDOMWindowInner::GetOrCreateWebIdentityHandler() {
-  if (mWebIdentityHandler) {
-    return mWebIdentityHandler;
-  }
-  mWebIdentityHandler = new WebIdentityHandler(this);
-  bool success = mWebIdentityHandler->MaybeCreateActor();
-  if (!success) {
-    mWebIdentityHandler = nullptr;
-  }
-  return mWebIdentityHandler;
-}
-
 CloseWatcherManager* nsPIDOMWindowInner::EnsureCloseWatcherManager() {
   if (!mCloseWatcherManager) {
     mCloseWatcherManager = new CloseWatcherManager();